core/test/variable_operations_test.cpp

```diff
@@ -1227,12 +1227,37 @@ TEST(VariableTest, nan_to_num_throws_when_input_and_replace_types_differ) {
 }
 
 TEST(VariableTest, nan_to_num) {
```

Comment:
Is it worth moving testing of these functions to the new pattern? Otherwise we have to touch this code again later. See refactoring also @DanNixon has started.